<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="margin">
     <number>6</number>
    </property>
    <item>
     <widget class="QLineEdit" name="lineEditHost">
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditName">
      <property name="text">
       <string>Ufonzak</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxPort">
      <property name="minimum">
       <number>1000</number>
      </property>
      <property name="maximum">
       <number>65536</number>
      </property>
      <property name="value">
       <number>9876</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowserLog"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonConnect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
